compiler/annotation/mod.rs

```diff
@@ -78,6 +78,18 @@ typedb_error!(
             "Error analysing return reduction.",
             ( typedb_source : Box<AnnotationError> )
         ),
+        CouldNotResolveReturnType(
```

Comment:
You can definitely reorder them if you want? But error 2 is different (to do with call arguments)?